init-script

```diff
@@ -0,0 +1,149 @@
+      printf "%s\n" "$text" >> "$bashrc"
+    else
+      printf "\n%s\n" "$text" >> "$bashrc"
+    fi
+  fi
+}
+
+trap 'ret=$?; test $ret -ne 0 && printf "failed\n\n" >&2; exit $ret' EXIT
+
+set -e
+
+if [ ! -d "$HOME/.bin/" ]; then
+  mkdir "$HOME/.bin"
+fi
+
+if [ ! -f "$HOME/.bashrc" ]; then
+  touch "$HOME/.bashrc"
+fi
+
+# shellcheck disable=SC2016
+append_to_bash_profile 'export PATH="$HOME/.bin:$PATH"'
+
+case "$SHELL" in
+  */bash) : ;;
+  *)
+    fancy_echo "Changing your shell to bash ..."
+      chsh -s "$(which bash)"
+    ;;
+esac
+
+brew_install_or_upgrade() {
+  if brew_is_installed "$1"; then
+    if brew_is_upgradable "$1"; then
+      fancy_echo "Upgrading %s ..." "$1"
+      brew upgrade "$@"
+    else
+      fancy_echo "Already using the latest version of %s. Skipping ..." "$1"
+    fi
+  else
+    fancy_echo "Installing %s ..." "$1"
+    brew install "$@"
+  fi
+}
+
+brew_is_installed() {
+  local name="$(brew_expand_alias "$1")"
+
+  brew list -1 | grep -Fqx "$name"
+}
+
+brew_is_upgradable() {
+  local name="$(brew_expand_alias "$1")"
+
+  ! brew outdated --quiet "$name" >/dev/null
+}
+
+brew_tap() {
+  brew tap "$1" 2> /dev/null
+}
+
+brew_expand_alias() {
+  brew info "$1" 2>/dev/null | head -1 | awk '{gsub(/:/, ""); print $1}'
+}
+
+brew_launchctl_restart() {
+  local name="$(brew_expand_alias "$1")"
+  local domain="homebrew.mxcl.$name"
+  local plist="$domain.plist"
+
+  fancy_echo "Restarting %s ..." "$1"
+  mkdir -p "$HOME/Library/LaunchAgents"
+  ln -sfv "/usr/local/opt/$name/$plist" "$HOME/Library/LaunchAgents"
+
+  if launchctl list | grep -Fq "$domain"; then
+    launchctl unload "$HOME/Library/LaunchAgents/$plist" >/dev/null
+  fi
+  launchctl load "$HOME/Library/LaunchAgents/$plist" >/dev/null
+}
+
+gem_install_or_update() {
+  if gem list "$1" --installed > /dev/null; then
+    fancy_echo "Updating %s ..." "$1"
+    gem update "$@"
+  else
+    fancy_echo "Installing %s ..." "$1"
+    gem install "$@"
+  fi
+}
+
+if ! command -v brew >/dev/null; then
+  fancy_echo "Installing Homebrew ..."
+    curl -fsS \
+      'https://raw.githubusercontent.com/Homebrew/install/master/install' | ruby
+
+    append_to_bash_profile '# recommended by brew doctor'
+
+    # shellcheck disable=SC2016
+    append_to_bash_profile 'export PATH="/usr/local/bin:$PATH"' 1
+
+    export PATH="/usr/local/bin:$PATH"
+else
+  fancy_echo "Homebrew already installed. Skipping ..."
+fi
+
+fancy_echo "Updating Homebrew formulas ..."
+brew update
+
+brew_install_or_upgrade 'git'
+brew_install_or_upgrade 'gdrive'
+
+brew unlink openssl && brew link openssl --force
+
+fancy_echo "Installing Java(8) Sdk"
+  brew cask install caskroom/versions/java8
+
+
+ruby_version="$(curl -sSL http://ruby.thoughtbot.com/latest)"
+
+\curl -L https://get.rvm.io | bash -s stable
+
+. ~/.bash_profile
+
+rvm get stable --auto-dotfiles
+
+# brew install node@8.9.1 <- Please install this node version (from the included pkg file) for the stability at this time.
```

Comment:
Installing with `.pkg` doesn't have enough permission to install `appium` 😅

Reply:
hmmm after installing node via the pkg I was being able to install appium via `npm install -g appium@1.7.1` @Dekablade01 could you take a look at a better way we could achieve it? I'd suggest to opt this out for this PR and initialize a different script for this because iOS is using different appium# than android for the sake of stability.

Reply:
Note! 
So, could you please rename this script for `android` specific. I'm going to add a new one for `iOS` in next pull request.